IngameNetManager.cs

```diff
@@ -1444,6 +1455,11 @@ void GrabNextPacket()
                                         targetActor.KillSilently();
                                     else
                                         targetActor.Kill(damage_info);
+                                    
+                                    // Reset the animator Controller in case the PerformKick Coroutine is Interrupted
+                                    if (OldKickController == null)
+                                        return;
```

Comment:
break instead. There could potentially be more packets to process.

As a side note, Die() in the NetActorController is a more reliable way to process deaths if they are causing an animation issue locally. Maybe we could move the kicking stuff there in a future PR.

Reply:
The thing about the assets is because I was testing them and forgot to remove them from the solution, sry.

And about handling the reset of the animator I just did the thing about moving the reset to the actual Die method and works fine